languages/translations/es.json

```diff
@@ -302,48 +302,48 @@
                 "exit_destination": "Tome la salida {exit} en la izquierda hacia {destination}"
             },
             "slight right": {
-                "default": "Ve cuesta abajo en la derecha",
-                "name": "Ve cuesta abajo en la derecha en {way_name}",
-                "destination": "Ve cuesta abajo en la derecha hacia {destination}",
+                "default": "Tome la salida en la derecha",
+                "name": "Tome la salida en la derecha en {way_name}",
+                "destination": "Tome la salida en la derecha hacia {destination}",
                 "exit": "Tome la salida {exit} en la derecha",
                 "exit_destination": "Tome la salida {exit} en la derecha hacia {destination}"
             }
         },
         "on ramp": {
             "default": {
-                "default": "Ve cuesta abajo",
-                "name": "Ve cuesta abajo en {way_name}",
-                "destination": "Ve cuesta abajo hacia {destination}"
+                "default": "Tome la salida",
```

Comment:
`on ramp` messages are for freeway entrances, not exits. Is “salida” the right word for an entrance?

Reply:
Nice catch! updated the translation.